EBNF.Compiler/Compiler.cs

```diff
@@ -292,8 +325,10 @@ public static string[] CompileEbnf(string rawData, string namespaceName)
         /// <param name="keyNameFile">TThe path to the SNK file used to sign the assembly.</param>
         public static CompilerResults CreateDll(string[] code, string filePath, string keyNameFile = "")
         {
-            // Create code provider to compile the dll
-            CSharpCodeProvider codeProvider = new CSharpCodeProvider();
+            var providerOptions = new Dictionary<string, string>();
+            providerOptions.Add("CompilerVersion", "v3.5");
```

Comment:
Please clarify why this was necessary

Reply:
because the Regex class is not available in lower versions